Added conf option to scope wg hub to right netns

JOUNAIDSoufiane opened the pull request:
This change is to add the 'create_hub_in_root_netns' boolean option designed to scope the wireguard hub creation to the root network namespace if set to True. To set, add the option to the site's defaults.

Commit:
Added conf option to scope wg hub to right netns

This change is to add the 'create_hub_in_root_netns' boolean option
designed to scope the wireguard hub creation to the root network
namespace if set to True. To set, add the option to the site's defaults.

JOUNAIDSoufiane commented:
![](https://github.trello.services/images/mini-trello-icon.png) [Fix Chi@Edge Chi-in-a-box Wireguard hub creation in wrong namespace](https://trello.com/c/JDIXb2XU/3273-fix-chiedge-chi-in-a-box-wireguard-hub-creation-in-wrong-namespace)

msherman64 commented:
Note! Requires code/container from this PR
https://github.com/ChameleonCloud/networking-wireguard/pull/5

msherman64 commented on kolla/defaults.yml:
```diff
@@ -257,6 +257,7 @@ neutron_dev_plugins:
 
 # minimal config for neutron wireguard agent
 neutron_wireguard_hub_endpoint: "{{ kolla_external_vip_address }}"
+create_hub_in_root_netns: "{{ create_hub_in_root_netns }}"
```
this needs a default:

normally I would say to default it to `false` so that just deploying the container won't change behavior.
however, as we're the only ones deploying this, lets set it to `true` because it will save some steps.

Commit:
Changed the name of the hub scoping conf option

create_hub_in_root_netns to neutron_wireguard_create_hub_in_root_netns.
A boolean to be added to defaults.yaml

msherman64 reviewed (changes requested):
In the current version here, the ml2_conf.ini part is fine.

However, under `kolla/defaults.yml`, I suggest the following:

```
neutron_wireguard_create_hub_in_root_netns: True
```

JOUNAIDSoufiane commented:
Done

JOUNAIDSoufiane committed:
Merge branch 'stable/xena' into soufianej-hub-port-config
